### PR TITLE
Add KingSwordYT

### PR DESCRIPTION
**Pull request by tmagicturtle**
The ability needs a shorter shortDesc, but I'm not sure how exactly to word it.

### CODE REVIEW

**Comment by tmagicturtle** [comment]
Also, if the effects of the move need to go into a secondary:, I'm not entirely sure how to do that.

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -443,6 +443,27 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// KingSwordYT
+	bambookingdom: {
+		desc: "On switch-in, this Pokemon's Defense and Special Defense are raised by 1 stage. Pokemon using physical moves against this Pokemon lose 1/8 of their maximum HP. Pokemon using special moves against this Pokemon lose 1/16 of their maximum HP. Attacking moves have their priority set to 7.",
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]: S-seven priority attacks?!?!

File: data/mods/ssb/random-teams.ts [hunk 2]
```diff
@@ -174,6 +174,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Cozy Cuddle',
 		evs: {atk: 252, spd: 4, spe: 252}, nature: 'Jolly', shiny: 1,
 	},
+	KingSwordYT: {
+		species: 'Pangoro', ability: 'Bamboo Kingdom', item: 'Rocky Helmet', gender: 'M',
+		moves: ['Body Press', 'Spiky Shield', 'Shore Up'],
+		signatureMove: 'Clash of Pangoros',
+		evs: {hp: 252, atk: 4, def: 252}, nature: 'Impish', shiny: 1,
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		evs: {hp: 252, atk: 4, def: 252}, nature: 'Impish', shiny: true,
```

File: data/mods/ssb/abilities.ts (hunk 1 above)
Comment (tmagicturtle) [reply data/mods/ssb/abilities.ts]: Woops. -7.

File: data/mods/ssb/random-teams.ts (hunk 2 above)
Comment (tmagicturtle) [reply data/mods/ssb/random-teams.ts]: Nature: Impish
Gender: Male
**Shiny: Yes**

Comment (ghost) [reply data/mods/ssb/random-teams.ts]: Fixed

Comment (tmagicturtle) [reply data/mods/ssb/random-teams.ts]: Ah. Sorry

**Pull request merged**